docs/core/tools/dotnet-run.md

```diff
@@ -60,7 +60,10 @@ Builds and runs the app using the specified [framework](../../standard/framework
 
 `-p|--project <PATH>`
 
-Specifies the path to the project file. It can be a path to a [csproj](csproj.md) file or to a directory containing a [csproj](csproj.md) file. It defaults to current directory if not specified. 
+Specifies the path to the project file. It can be a path to a [csproj](csproj.md) file ~or to a directory containing a [csproj](csproj.md) file~ (See NOTE). It defaults to current directory if not specified.
```

Comment:
Removing the or clause makes the "can" inaccurate. I'd change this to "Specifies the path and name of the project file. (See the NOTE.)"
to current directory --> to the current directory